test/nginx_system_test.sh

```diff
@@ -252,7 +252,7 @@ fi
 
 # check_stat in system_test_helpers.sh needs to know whether statstistics are
 # enabled, which is always the case for ngx_pagespeed.
-statistics_enabled=1
+statistics_enabled=0
```

Comment:
This isn't a good fix for the statistics issue.  As far as I know only the cache flushing test has a problem, but here we disable all stats testing in ngx_pagespeed.